internal/kafka/kafka.go

```diff
@@ -39,7 +39,7 @@ var (
 
 func createSASLMechanism(saslMechanismName string, username string, password string) (sasl.Mechanism, error) {
 	switch strings.ToLower(saslMechanismName) {
-	case "plain":
+	case "plain", "none":
```

Comment:
Unrelated, noticed we use "none" in some code so added it. Not too much useful, blank value will do as well.